contrib/devtools/proto-tools-installer.sh

```diff
@@ -85,9 +85,13 @@ f_install_buf() {
 
 f_install_protoc_gen_gocosmos() {
   f_print_installing_with_padding protoc-gen-gocosmos
-  pushd "${TEMPDIR}" >/dev/null
+
+  if ! grep "github.com/gogo/protobuf => github.com/regen-network/protobuf" go.mod &>/dev/null ; then
+    echo -e "\tPlease run this command from the root folder."
```

Comment:
```suggestion
    echo -e "\tPlease run this command from somewhere inside the cosmos-sdk folder."
```

talked to @robert-zaremba, suggested wording change.